api/v3/views/protected_area.rabl

```diff
@@ -97,3 +97,15 @@ end
 if @current_user.access_to?(ProtectedArea, :owner_type)
   attribute :owner_type
 end
+
+if @current_user.access_to?(ProtectedArea, :pame_evaluations)
```

Comment:
It doesn't look like you defined permissions for the `pame_evaluations`.
You would need to add this to the `api_attributes` for the `ProtectedArea` object and then grant permissions to the user, otherwise you won't see this appearing in the resulting json

Reply:
Thanks @Levia I did not realise this, I had manually updated the permissions JSON object myself locally. I will need to update this!

Reply:
@Levia I have added this to the `api_attributes` for the `ProtectedArea` object but I am not sure how to grant the permissions to the user. I have studied the admin section of the API and I can see how it is done. I just wondered about the best way of adding the permissions whether on the console with irb or through a curl request?

Reply:
@drewpotter I've emailed you the other day about the admin interface.
You should be able to get there via `/admin` and then the credentials are in your env file